jupyterlab/labapp.py

```diff
@@ -41,6 +41,41 @@
 if version != app_version:
     version = '%s (dev), %s (app)' % (__version__, app_version)
 
+buildFailureMsg = """Build failed.
+Troubleshooting: If the build failed due to an out-of-memory error, you
+may be able to fix it by disabling the `dev_build` and/or `minimize` options.
+
+If you are building via the `build jupyter lab` command, you can disable
+these options like so:
+
+build jupyter lab --dev-mode=False --minimize=False
```

Comment:
`jupyter lab build`, right? If so, also above.

Reply:
Yep, I done goofed

Reply:
Fixed it